tests/exploits/cve_2024_6387_test.py

```diff
@@ -0,0 +1,88 @@
+from unittest import mock
+from unittest.mock import patch
+
+from agent import definitions
+from agent.exploits import cve_2024_6387
+
+
+@patch("agent.exploits.cve_2024_6387._get_ssh_banner")
```

Comment:
Since you are patching it, expose it `get_ssh_banner` instead of `_get_xxx`. Might as put it in a common folder for shared usage.